.vscode/settings.json

```diff
@@ -68,5 +68,10 @@
   ],
   // Linux
   "terminal.integrated.defaultProfile.linux": "zsh",
-  "python.REPL.enableREPLSmartSend": false
+  "python.REPL.enableREPLSmartSend": false,
+  "python.testing.pytestArgs": [
+    "django"
+  ],
+  "python.testing.unittestEnabled": false,
+  "python.testing.pytestEnabled": true
```

Comment:
Not sure why these changes are needed, but I assume they won't cause issues for others. If so, we can remove them in another PR.